org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -73,14 +73,14 @@ Parameters:
     Type: String
     Default: '906769aa66-6cdd2cc1-6d3d-49d8-b105-37c22bb3d402'
 
-  synapseAdminGroup:      #JC aws-synapse-admins
```

Comment:
please make sure this is removed from JC

Reply:
The user group is deleted in JC.